src/Build.OM.UnitTests/Definition/Project_Tests.cs

```diff
@@ -4082,6 +4084,49 @@ public void ProjectImportedEventInvalidFile()
         }
 
         [Fact]
+        public void ProjectImportedEventMissingFile()
```

Comment:
If it does not already exist, can you also add a test for a glob based import where the glob expands to nothing? In that case, the event should still be there, but `ImportedProjectFile` should be empty.

Reply:
Do you think `ImportIgnored` should be true in this case?

Reply:
I agree with you. Can you please add a comment about this on ImportIgnored?

Reply:
Done